tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -100,7 +96,7 @@ EvolutionSystem:
           Amplitude: 0.75            # 0.75 / (m_A + m_B)
           Width: 38.415              # 2.5 * (x_B - x_A)
           Center: [0.0, 0.0, 0.0]
-        FunctionOfTimeForScaling: ExpansionFactor
+        FunctionOfTimeForScaling: Expansion
```

Comment:
Self change request: Take this out of options and just hard code the name because I believe this can only be used with a binary compact object domain (i.e. with an expansion map) but I'm not sure.